final_prep/README.md

```diff
@@ -19,37 +19,37 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![image](https://user-images.githubusercontent.com/15107515/132791090-51e56807-1dea-499d-9d68-a63e9259c7fd.png)
```

Comment:
Nice job here- this is a good start to a routine. I say 'start', because a calendar is always a work in progress. A little feedback:
- I think if you are trying to do most of your weekend work on Sundays to leave Saturday more open, that should work a lot of the time, but perhaps sometimes better than others, depending on project partners. Also, it could prove hard to stay focused that long on a Sunday. You know yourself best, so I just encourage time put into evaluating how you are doing with that schedule, and adjust as needed.
- You might think about scheduling blocks over your project/study time to indicate how long you plan on spending on a particular task. The trick is to avoid context switching too frequently during a longer study block. A lot of time can be lost switching from one thing to another frequently.